By default tests are run on build and install. Normally, Alpine
tests are invoked separately in a check stage. To allow doing
that this patch disable the automatic test invocations during build and
install.

diff -upr bmake.orig/boot-strap bmake/boot-strap
--- a/boot-strap	2024-03-15 11:26:18.664828810 +0100
+++ b/boot-strap	2024-03-15 11:27:12.081555010 +0100
@@ -435,9 +435,6 @@ op_build() {
 	[ -s make-bootstrap.sh ] || op_configure
 	chmod 755 make-bootstrap.sh || exit 1
 	./make-bootstrap.sh || exit 1
-	case "$op" in
-	build) rm -f tested; op_test;;
-	esac
 }
 
 op_test() {
@@ -461,7 +458,6 @@ op_clean() {
 }
 
 op_install() {
-	op_test
 	case "$INSTALL_PREFIX,$INSTALL_BIN,$prefix" in
 	,$HOST_TARGET/bin,*/$HOST_TARGET)
 		INSTALL_PREFIX=`dirname $prefix`
